src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -906,6 +907,10 @@ namespace ReportSizingManager {
 									TotCapTempModFac = CurveValue ( DataTotCapCurveIndex, CoilInWetBulb, OutTemp );
 								} else {
 									TotCapTempModFac = 1.0;
+									
+									if( SameString( CompType, "Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl" ) ){
+										EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, TotCapTempModFac);
```

Comment:
Move this call to DXCoils, and set a new variable to use in RequestSizing. e.g. DataTotCapCurveValue 

		if ( DataTotCapCurveIndex > 0 ) {
			TotCapTempModFac = CurveValue ( DataTotCapCurveIndex, CoilInWetBulb, OutTemp );
		} else if ( DataTotCapCurveValue > 0.0 ) {
			TotCapTempModFac = DataTotCapCurveValue;
		} else {
			TotCapTempModFac = 1.0;
		}


DXCoils:

	if ( DXCoil( DXCoilNum ).DXCoilType_Num == CoilDX_CoolingTwoStageWHumControl ) {
		SizingMethod = CoolingCapacitySizing;
		CompName = DXCoil( DXCoilNum ).Name + ":" + DXCoil( DXCoilNum ).CoilPerformanceName( Mode );
		FieldNum = 1;
		TempSize = DXCoil( DXCoilNum ).RatedTotCap( Mode );
		SizingString = DXCoilNumericFields( DXCoilNum ).PerfMode( Mode ).FieldNames( FieldNum ) + " [W]";
	} else if ( DXCoil( DXCoilNum ).DXCoilType_Num == CoilDX_HeatingEmpirical || DXCoil( DXCoilNum ).DXCoilType_Num == CoilVRF_Heating || DXCoil( DXCoilNum ).DXCoilType_Num == CoilVRF_FluidTCtrl_Heating ) {
		SizingMethod = HeatingCapacitySizing;
		CompName = DXCoil( DXCoilNum ).Name;
		FieldNum = 1;
		TempSize = DXCoil( DXCoilNum ).RatedTotCap( Mode );
		if ( CoilVRF_Heating || DXCoil( DXCoilNum ).DXCoilType_Num == CoilVRF_FluidTCtrl_Heating ) {
				EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, DataTotCapCurveValue);
		}
		SizingString = DXCoilNumericFields( DXCoilNum ).PerfMode( Mode ).FieldNames( FieldNum ) + " [W]";
		DataCoolCoilCap = DXCoolCap;
	} else if ( DXCoil( DXCoilNum ).DXCoilType_Num == CoilDX_HeatPumpWaterHeaterPumped || DXCoil( DXCoilNum ).DXCoilType_Num == CoilDX_HeatPumpWaterHeaterWrapped ) {
		SizingMethod = CoolingCapacitySizing;
		CompName = DXCoil( DXCoilNum ).Name;
		FieldNum = 1;
		TempSize = DXCoil( DXCoilNum ).RatedTotCap( Mode );
		SizingString = DXCoilNumericFields( DXCoilNum ).PerfMode( Mode ).FieldNames( FieldNum ) + " [W]";
		PrintFlag = false;
	} else {
		SizingMethod = CoolingCapacitySizing;
		CompName = DXCoil( DXCoilNum ).Name;
		FieldNum = 1;
		TempSize = DXCoil( DXCoilNum ).RatedTotCap( Mode );
		SizingString = DXCoilNumericFields( DXCoilNum ).PerfMode( Mode ).FieldNames( FieldNum ) + " [W]";
	}
	CompType = DXCoil( DXCoilNum ).DXCoilType;
	DataIsDXCoil = true;
	DataTotCapCurveIndex = DXCoil ( DXCoilNum ).CCapFTemp ( Mode );
	DataEMSOverrideON = DXCoil( DXCoilNum ).RatedTotCapEMSOverrideOn( Mode );
	DataEMSOverride = DXCoil( DXCoilNum ).RatedTotCapEMSOverrideValue( Mode );
	RequestSizing( CompType, CompName, SizingMethod, SizingString, TempSize, PrintFlag, RoutineName );
	// be sure to reset to 0 after call to request sizing
	DataTotCapCurveValue = 0.0;